extensions/vertx-graphql/deployment/src/test/java/io/quarkus/vertx/graphql/deployment/ErroneousConfigTest.java

```diff
@@ -0,0 +1,25 @@
+package io.quarkus.vertx.graphql.deployment;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.deployment.configuration.ConfigurationError;
+import io.quarkus.test.QuarkusUnitTest;
+
+public class ErroneousConfigTest {
+
+    @RegisterExtension
+    static final QuarkusUnitTest config = new QuarkusUnitTest()
+            .setExpectedException(ConfigurationError.class)
+            .setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class)
+                    .addAsResource(new StringAsset("quarkus.vertx-graphql-ui.path=/\n"), "application.properties"));
+
+    @Test
+    public void shouldNotStartApplicationIfPathIsASlash() {
+        Assertions.fail();
```

Comment:
I don't understand how this test works.

Reply:
Specifying a `/` results in  build failure - https://github.com/quarkusio/quarkus/pull/5253/files#diff-2e471b456c7129680f3d600014449819R17 so basically this test method will never be called.

Reply:
Oh I see!